- polytunnel:
  - nx: 10     # polytunnel 1
    ny: 5
    dx: 3.01
    dy: 1.35
    xpose: 5
    ypose: 1.35
  - nx: 10     # polytunnel 2
    ny: 5
    dx: 3.01
    dy: 1.35
    xpose: 5
    ypose: -6.75

- riseholme_enclosure:
  - xpose: 8.3465     
    ypose: -2.2
